apps/angular/pipe-intermediate/src/app/wrapper.pipe.ts

```diff
@@ -0,0 +1,21 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'showName',
+  standalone: true,
+})
+export class ShowNamePipe implements PipeTransform {
+  transform(name: string, index: number): string {
+    return `${name} - ${index}`;
+  }
+}
+
+@Pipe({
+  name: 'isAllowed',
+  standalone: true,
+})
+export class IsAllowedPipe implements PipeTransform {
+  transform(age: number, isFirst: boolean): string {
+    return isFirst ? 'always allowed' : age > 25 ? 'allowed' : 'declined';
+  }
+}
```

Comment:
The goal is to create a pipe that wrap the function inside the use the memoization of the pipe